pkg/cli/pit/format.go

```diff
@@ -37,18 +36,21 @@ import (
 	"github.com/spf13/viper"
 )
 
-func formatCommand() *cobra.Command {
+// WriteLiveCDFunc is a type for the writeLiveCD function to allow for dependency injection in testing
+type WriteLiveCDFunc func(device string, iso string, size string) error
+
+func formatCommand(formatFunc WriteLiveCDFunc) *cobra.Command {
 	c := &cobra.Command{
 		Use:   "format DISK ISO SIZE",
 		Short: "Formats a disk as a LiveCD",
 		Long:  `Formats a disk as a LiveCD using an ISO.`, // ValidArgs: []string{"disk", "iso", "size"},
 		Args:  cobra.ExactArgs(3),
 		Run: func(c *cobra.Command, args []string) {
```

Comment:
The error from `formatFunc` isn't checked.

Since it's the last function called, perhaps we should just return its error to `RunE`
```suggestion
		RunE: func(c *cobra.Command, args []string) error {
```

Then on line 53:
```go
err := formatFunc(device, iso, size)
return err
```